docker/main/run.sh.example

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
+
+docker run --rm -it \
+        -e S3_SUBMISSION_ARCHIVE='' \
+        -e S3_VALIDATION_BUCKET='' \
+        -e AWS_BATCH_JOB_ID='' \
+        -e AWS_BATCH_JOB_NODE_INDEX= '' \
+        -e MAIN_SLEEP_INTERVAL= '' \
+	-e WORKER_INIT_TIMEOUT= '' \
+	-e AWS_SNS_TOPIC_ARN= '' \
+	-e DEBUG= '' \
```

Comment:
You might want to remove the extra space between the = and the ' for AWS_BATCH_JOB_NODE_INDEX and other values that have that extra space.